scripts/tests/cirque_tests.sh

```diff
@@ -56,7 +56,7 @@ function __cirquetest_start_flask() {
     echo 'Start Flask'
     cd "$REPO_DIR"/third_party/cirque/repo
     FLASK_APP='cirque/restservice/service.py' \
-        PATH="$PATH":"$REPO_DIR"/third_party/cirque/repo/openthread/output/x86_64-unknown-linux-gnu/bin/ \
+        PATH="$PATH":"$REPO_DIR"/third_party/cirque/repo/openthread/output/simulation/bin/ \
```

Comment:
Change seems unrelated. Could you merge with master (likely this is already in master)